components/TextFields/styling/README.md

```diff
@@ -4,6 +4,8 @@ MDC's text fields are designed to be styled by controllers implementing the `MDC
 
 Some of the classes are marked as "Preferred". These are known, through extensive UX research, to have big performance gains in usability.
 
+There's also a class called `MDCTextInputControllerBase`. It's meant to be used as a super-class for controllers that want to get floating placeholder behavior asap. Several other controllers inherit from it.
```

Comment:
Super-nit: Please replace "asap" with prose.  "...controllers that want default floating placeholder behavior with minimal customization."   I think it will make it clearer what the "base" controller helps to optimize.

Reply:
Done.